.github/workflows/ios.yml

```diff
@@ -16,6 +16,7 @@ jobs:
         with:
           fetch-depth: 2
       - uses: actions/setup-node@v3
+      - run: npm install -g yarn
```

Comment:
`yarn` comes installed with default path in `actions/setup-node` !

https://github.com/actions/setup-node/blob/d1b197b965eb46e1a1878caad41c4ab3efae133f/src/main.ts#L109